src/Pagerfanta/View/DefaultView.php

```diff
@@ -238,64 +280,3 @@ public function getName()
         return 'default';
     }
 }
-
-/*
-
-CSS:
-
-.pagerfanta {
-}
-
-.pagerfanta a,
-.pagerfanta span {
-    display: inline-block;
-    border: 1px solid blue;
-    color: blue;
-    margin-right: .2em;
-    padding: .25em .35em;
-}
-
-.pagerfanta a {
-    text-decoration: none;
-}
-
-.pagerfanta a:hover {
-    background: #ccf;
-}
-
-.pagerfanta .dots {
-    border-width: 0;
-}
-
-.pagerfanta .current {
-    background: #ccf;
-    font-weight: bold;
-}
-
-.pagerfanta .disabled {
-    border-color: #ccf;
-    color: #ccf;
-}
-
-COLORS:
-
-.pagerfanta a,
-.pagerfanta span {
-    border-color: blue;
-    color: blue;
-}
-
-.pagerfanta a:hover {
-    background: #ccf;
-}
-
-.pagerfanta .current {
-    background: #ccf;
-}
-
-.pagerfanta .disabled {
-    border-color: #ccf;
-    color: #cf;
-}
-
-*/
```

Comment:
Why is there some css rules in this php file?
If it's useful for the implementation, why not put it in the readme? Or just add comments above to explain why it's here?